base/src/main/java/io/fabric8/launcher/base/identity/Identity.java

```diff
@@ -14,14 +14,30 @@
  */
 public interface Identity extends Consumer<IdentityVisitor> {
 
-    @Value.Lazy
-    default String toRequestHeaderKey() {
-        return createRequestAuthorizationHeaderKey(this);
-    }
+    /**
+     * Get the standard authorization type to use with this identity
+     *
+     * @return the {@link AuthorizationType}
+     */
+    AuthorizationType getStandardAuthorizationType();
```

Comment:
Why not just getAuthorization?

Reply:
if I change to `getDefaultAuthorizationType` and `toRequestAuthorization()` do you prefer?

Reply:
My point is to why add the `Default` or `Standard` in this method name.

Reply:
because it's just one of the available authorization type to use as standard/default, meaning when you make an http request with a token, bearer is the most common usage and when you user password, basic is.

Reply:
Ah so better use `Default` instead of `Standard` and add a @Derived or the generated Immutable will be wrong I guess

Reply:
Done ;)